File: algorithms/warmup/compareTheTriplets.pl
#!/usr/bin/perl

use warnings;
use strict;

my $aMarkLine = <STDIN>;
my @a = split(' ', $aMarkLine);

my $bMarkLine = <STDIN>;
my @b = split(' ', $bMarkLine);

my $aTotal = 0;
my $bTotal = 0;

for (my $i = 0; $i < 3; $i++) {
    if ($a[$i] > $b[$i]) {
        $aTotal++;
    } elsif ($b[$i] > $a[$i]) {
        $bTotal++;
    }
}

print "$aTotal $bTotal";

File: algorithms/warmup/compareTheTriplets.py
#!/usr/bin/env python3

aMarkList = input().strip().split(' ')
bMarkList = input().strip().split(' ')

# print(aMarkList, bMarkList)
aTotal = 0
bTotal = 0

for i in range(3):
    if int(aMarkList[i]) > int(bMarkList[i]):
        aTotal += 1
    elif int(bMarkList[i]) > int(aMarkList[i]):
        bTotal += 1
    else:
        pass

print(aTotal, bTotal)
